apps/frontend/src/app/Data/Characters/Kaveh/index.tsx

```diff
@@ -427,8 +412,24 @@ const sheet: ICharacterSheet = {
     constellation5: ct.talentTem('constellation5', [
       { fields: [{ node: skillC5 }] },
     ]),
-    constellation6: ct.talentTem('constellation6'),
-  },
+    constellation6: ct.talentTem('constellation6', [
+      ct.fieldsTem('constellation6', {
+        canShow: equal(condAfterBurst, 'on', 1),
+        fields: [
+          {
+            node: infoMut(dmgFormulas.constellation6.dmg, {
```

Comment:
actually on second thought, this shouldn't be behind a conditional at all

Reply:
yeah i thought that too but since it used to be... will change